### PR TITLE
Fixes ci-ingress-gce-upgrade-e2e

### PR DESCRIPTION
**Pull request by rramkumar1**
**What this PR does / why we need it**:
This PR fixes the broken ci-ingress-gce-upgrade-e2e job. The issue was that the ingress was being deleted by the test framework before the upgraded ingress could properly sync. Therefore, the resources were never cleaned up, which caused test failure. 

```release-note
None
```

cc @MrHohn 
/assign @bowei

### CODE REVIEW

File: test/e2e/upgrades/ingress.go [hunk 1]
```diff
@@ -93,10 +97,11 @@ func (t *IngressUpgradeTest) Setup(f *framework.Framework) {
 		"kubernetes.io/ingress.global-static-ip-name": t.ipName,
 		"kubernetes.io/ingress.allow-http":            "false",
 	}, map[string]string{})
-	jig.AddHTTPS("tls-secret", "ingress.test.com")
+	framework.DescribeIng(jig.Ingress.Namespace)
```
Comment (MrHohn) [inline test/e2e/upgrades/ingress.go]: Do we still need this describe? Or if we are keeping it, move it to after `AddHTTPS()`?

**Review by MrHohn — APPROVED** [review]
Just a minor comment.

/lgtm

File: test/e2e/upgrades/ingress.go (hunk 1 above)
Comment (rramkumar1) [reply test/e2e/upgrades/ingress.go]: Removed.

**Pull request merged**